### PR TITLE
PLANET-7070 Re-write the Page Layouts Modal test in Playwright

### PR DESCRIPTION
**Pull request by Osong-Michael**
**Description: [PLANET-7070](https://jira.greenpeace.org/browse/PLANET-7070)**

**Testing:** 
 Running `npx playwright test tests/e2e/page-layouts-modal.spec.js` on your terminal for the plugin should run the test and validate them.

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [janus](https://www-dev.greenpeace.org/test-janus) | [admin](https://www-dev.greenpeace.org/test-janus/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-janus/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-janus) | [composer-local.json](https://github.com/greenpeace/planet4-test-janus/blob/main/composer-local.json)

:watch: 2023.04.17 13:06:18

File: tests/e2e/page-layouts-modal.spec.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import {login} from './tools/lib/login';
+
+test('checks if the welcome modal on the editor is present and closed when the button is clicked', async ({page, context}) => {
+  test.setTimeout(240 * 1000);
+  await page.goto('./');
+  await login(page, context);
+  await page.goto('./wp-admin/post-new.php?post_type=page');
+  const locator = page.locator('.components-modal__screen-overlay');
+  await expect(locator).toBeVisible();
+  await page.getByRole('button', {name: 'Close dialog'}).click();
```
Comment (GP-Dan-Tovbein) [inline tests/e2e/page-layouts-modal.spec.js:11]: I tested locally and it failed here. Thus, I changed to `await page.getByRole('button', {name: 'Close'}).click();` and it finally passed.

Screenshot 2023-05-03 at 08 51 39


What do you think?

File: tests/e2e/page-layouts-modal.spec.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const {test, expect} = require('@playwright/test');
+import {login} from './tools/lib/login';
+
+test('checks if the welcome modal on the editor is present and closed when the button is clicked', async ({page, context}) => {
+  test.setTimeout(240 * 1000);
```
Comment (GP-Dan-Tovbein) [inline tests/e2e/page-layouts-modal.spec.js:5]: Feel free to remove this `timeout` if you think it's not really necessary through this `spec`

**Review by GP-Dan-Tovbein — APPROVED** [review]
Code looks good to me! I've tested locally and worked perfectly. Nice job 👏 

I've just left a small suggestion of a change but it's up to you to make it.

**Pull request merged**